api/room/schema.py

```diff
@@ -79,8 +91,14 @@ def mutate(self, info, office_id, **kwargs):
         # remove block ID from kwargs, can't be saved in rooms model
         if kwargs.get('block_id'):
             kwargs.pop('block_id')
+        # pop the room_tags kwargs to create the room object
+        # this is because it populates the association table.
+        room_tags = ''
```

Comment:
Hello @bevkololi, I think for the room_tag you should assign it to an empty list so that it id intuitive that the variable expects to be populated with a list of items.

Reply:
Thank you for that @caveinn, working on it.

Reply:
This has been updated @caveinn